consensus/pacemaker.go

```diff
@@ -262,3 +267,19 @@ func (p *paceMaker) getStepTimeout(round uint64) timePkg.Duration {
 	baseTimeout := timePkg.Duration(int64(timePkg.Millisecond) * int64(p.pacemakerCfg.GetTimeoutMsec()))
 	return baseTimeout
 }
+
+func (p *paceMaker) onBeforeNewHeight() {
```

Comment:
Linking to our discussion here: https://github.com/pokt-network/pocket/pull/374#discussion_r1046583124

If we do decide to keep it, we should really leave a comment explaining "Not used, but can be used as a hook", etc..